daemons/dss-index/app.py

```diff
@@ -11,4 +13,5 @@
 
 @app.s3_event_handler(bucket=s3_bucket, events=["s3:ObjectCreated:*"])
 def dispatch_indexer_event(event, context) -> None:
+    app.log.setLevel(logging.DEBUG)
```

Comment:
Yes, for now I want all logs down to the debug level from the sync and indexer daemons to show up in cloudwatch.